components/Layout.js

```diff
@@ -10,6 +11,7 @@ export default function Layout({ children }) {
         <h1>Bloom Buddy</h1>
       </header>
       <main>{children}</main>
+      <NavBar></NavBar>
```

Comment:
Use a self-closing tag here since `NavBar` has no children. 

```suggestion
      <NavBar />
```